wazuh/wazuh-dashboard-plugins
# Some links in the threat hunting dashboard table do not redirect or have no links

### Description

This PR fix the issue some links in the threat hunting dashboard table do not redirect or have no links
 
### Issues Resolved

#6830 

### Evidence

https://github.com/user-attachments/assets/0b49c93b-7361-4f32-bd7d-67df8042c1d7

### Test

- Go to the Threat Hunting tab, in the dashboard table click on one of the values in the `rule.mitre.id` field and check that it redirects correctly.

- Go to the Threat Hunting tab, in the dashboard table click on one of the values in the `rule.mitre_techniques` field and check that it redirects correctly.

### Check List
